### PR TITLE
Init: casbin-nginx-module

### PR DESCRIPTION
**Pull request by sheny1xuan**

### CODE REVIEW

File: NginxCasbinConf.hpp [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/*
```
Comment (hsluoyz) [inline NginxCasbinConf.hpp:1]: Use the apache header of Casbin-CPP repo

File: NginxCasbinConf.hpp [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    NgxCasbinConf() = default;
+    ~NgxCasbinConf() = default;
+
+    enum class ADOPTER_TYPE { 
+        FILE_ADOPTER,
+        MYSQL_ADOPTER,
+        UNKNOW_TYPE,
+    };
+
+    // 一定要初始化
```
Comment (hsluoyz) [inline NginxCasbinConf.hpp:29]: Use English everywhere

Comment (sheny1xuan) [reply NginxCasbinConf.hpp]: Fix it, and I will be careful latter.

**Comment by hsluoyz** [comment]
@sheny1xuan remove these commits:

image

**Comment by hsluoyz** [comment]
@EmperorYP7 thanks!

**Pull request merged**